packages/office-ui-fabric-react/src/components/Button/BaseButton.tsx

```diff
@@ -470,7 +470,7 @@ export class BaseButton extends BaseComponent<IBaseButtonProps, IBaseButtonState
     let { keytipProps } = this.props;
 
     const classNames = getSplitButtonClassNames
-      ? getSplitButtonClassNames(!!disabled, !!allowDisabledFocus, !!this.state.menuProps, !!checked)
+      ? getSplitButtonClassNames(!!disabled, !!this.state.menuProps, !!checked, !!allowDisabledFocus)
```

Comment:
Do we want to change the order of this?

I'm looking at the parameter of getSplitButtonClassNames and here's what it looks like:

(styles: IButtonStyles, disabled: boolean, expanded: boolean, checked: boolean)

Reply:
Yes we do, it was changed in the most recent version of fabric (6.13.0) and it broke the existing callback parameter ordering resulting in them getting processed differently.

The creator can leverage whatever parameters they want from the signature, putting it at the end doesn't change the ordering and allows creators who want to leverage the new parameter to do so by updating their callback